src/test/java/pl/net/was/JsonTrinoConverterTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package pl.net.was;
+
+import com.fasterxml.jackson.databind.node.TextNode;
+import io.swagger.v3.oas.models.media.Schema;
+import io.trino.spi.type.TimestampType;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class JsonTrinoConverterTest
+{
+    @ParameterizedTest
+    @ValueSource(strings = {
+            "2024-09-24T19:17:58.638+04:00",
+            "2024-09-24T18:30:47.94+04:00"
+    })
+    void shouldConvertDifferentDateTimeFormatsIntoLong(String dateTime)
+    {
+        var node = TextNode.valueOf(dateTime);
+        var type = TimestampType.createTimestampType(3);
+        var schema = new Schema<>();
+        schema.setFormat("date-time");
+
+        assertThat(JsonTrinoConverter.convert(node, type, schema)).isInstanceOf(Long.class);
+    }
```

Comment:
I agree, no need to make it a parametrized test. Also assert the actual value of the conversion result, not just it type.
```suggestion
    void shouldConvertDifferentDateTimeFormatsIntoLong()
    {
        TimestampType type = TimestampType.createTimestampType(3);
        Schema schema = new Schema<>().setFormat("date-time");

        assertThat(JsonTrinoConverter.convert(TextNode.valueOf("2024-09-24T19:17:58.638+04:00"), type, schema)).isInstanceOf(Long.class);
        assertThat(JsonTrinoConverter.convert(TextNode.valueOf("2024-09-24T18:30:47.94+04:00"), type, schema)).isInstanceOf(Long.class);
    }
```